README.md

```diff
@@ -0,0 +1,56 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Create Enumerable
+
+> This project uses Ruby to create Enumerable.
+
+
+## Built With
+
+- Ruby
+- git
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+Clone the project by using terminal:
+
+```
```

Comment:
- [ ] [OPTIONAL] Due to these 3 tilde signs everything below is put in a code block, if you remove these the styling in your readme file will look a lot better.